src/controllers/touristCenterController.js

```diff
@@ -0,0 +1,120 @@
+import db from "../services/AdminServices/touristCenterService";
+import { validation, validateId } from "../validation/touristCenterValidation";
+
+/**
+ * @class touristCenterController
+ * @description create, read, update & delete touristcenter
+ * @exports touristCenterController
+ */
+export default class touristCenterController {
+  /**
+   * @param {object} req - The user request object
+   * @param {object} res - The user response object
+   * @returns {object} Success message
+   */
+  static async addTouristCenter(req, res) {
+    try {
+      const {
+        countryId, gallery, name, location, about
```

Comment:
I think in creating a tourist center, countryName instead of countryId should be passed inside the req.body. It'll be easier to know the country whose tourist center you want to create with the countryName than countryId(in which case you have to query the database first to get the countryId)
Every other thing looks fine